### DISCUSSION TITLE
Import via SFTP

### DISCUSSION THREAD

**Discussion by twiesing** [immich-app/immich#8988] [Q&A]
I currently use Photoprism and plan to switch to Immich. I am actually importing my photos from various devices via https://www.photosync-app.com, which I would like to keep. Where should I store the images (via SFTP) so that Immich automatically recognizes and indexes them? (I don't want an external library)

**Comment by mmomjian** [comment]
There is no auto import folder. You can SFTP them over to a folder of your choosing then import using Immich CLI or Immich-Go. You could setup a cron job to import photos from a folder then delete or move them.

**Comment by Demian98** [comment]
https://github.com/immich-app/immich/discussions/20522 Seems like this could solve it
